node/silkworm/downloader/sentry_client.cpp

```diff
@@ -18,11 +18,72 @@
 
 #include <silkworm/common/log.hpp>
 
+#include "rpc/SetStatus.hpp"
+#include "rpc/ReceiveMessages.hpp"
+
 namespace silkworm {
 
 SentryClient::SentryClient(std::string sentry_addr)
     : base_t(grpc::CreateChannel(sentry_addr, grpc::InsecureChannelCredentials())) {}
 
 void SentryClient::exec_remotely(SentryRpc& rpc) { base_t::exec_remotely(rpc); }
```

Comment:
`exec_remotely` is not virtual in `rpc::Client`; I don't see a point in defining it in `SentryClient`.

Reply:
Yes. See also the same [comment in previous PR](https://github.com/torquem-ch/silkworm/pull/483#issuecomment-965450396)

Reply:
The point is this: exec_remotely() is the most important method of the SentryClient so it's nice to see it in the SentryClient class declaration, for completeness and for easy of reading. Also the comment on the method completes the picture. 

But, anyway, I solved replacing the method redefinition with this directive:

`using base_t::exec_remotely; // exec_remotely(SentryRpc& rpc) sends a rpc request to the remote sentry`